bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -548,21 +548,21 @@ private Sitemap getSitemap(String sitemapname) {
     }
 
     private boolean blockUnlessChangeOccurs(String sitemapname, String pageId) {
-        boolean timeout = false;
+        boolean changeOccurred = true;
```

Comment:
Is it correct to claim that changeOccurred if there is no sitemap with the given name?

Reply:
If I set it to false, the caller will consider that the timeout was reached which is wrong in the case you mention.
Maybe I should reverse the logic so that this method returns true when the timeout is reached.

Reply:
Might be easier to read then - up to you.